src/main/java/core/basesyntax/service/impl/DeepCopyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DeepCopy;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeepCopyImpl implements DeepCopy {
```

Comment:
It's good practice to document your classes and methods. Consider adding a JavaDoc comment to explain the purpose of this class and its method.